src/onchain.rs

```diff
@@ -24,7 +24,7 @@ pub async fn pay_onchain(
     payload: OnchainRequest,
 ) -> anyhow::Result<String> {
     if payload.sats > MAX_SEND_AMOUNT {
-        anyhow::bail!("max amount is 1,000,000");
+        anyhow::bail!("max amount is 10,000,000");
```

Comment:
Missed a spot in https://github.com/MutinyWallet/mutinynet-faucet-rs/commit/4288756687b4baef2a0cbda35d148785025cc7ae 👀